css/markdown-editor.styl

```diff
@@ -29,6 +29,15 @@
   right: 0
   top: 0
 
+  h1
+    color: BURNT_ORANGE
```

Comment:
This is being applied to the markdown editor across the whole project builder now. If it's difficult to scope this just for the editor for the about pages, we could alternatively add info about the style in the instructions area for the about page editor instead.